qcodes/monitor/monitor.py

```diff
@@ -40,7 +40,8 @@
 from qcodes.instrument.parameter import Parameter
 
 
-def _get_all_tasks():
+def _get_all_tasks() -> Callable[[Optional[asyncio.AbstractEventLoop]],
+                                 Set[asyncio.Task]]:
```

Comment:
This now trips up python 3.6. We should probably rewrite this with `sys.version_info` as suggested by the mypy docs https://mypy.readthedocs.io/en/stable/common_issues.html#python-version-and-system-platform-checks